### DISCUSSION TITLE
Projet fin d’étude

### DISCUSSION THREAD

**Discussion by Saifoulaye** [Security-Onion-Solutions/securityonion#13026] [Unsupported Versions]
Bonsoir tout le monde, je réponds au nom de Diallo Saifoulaye étudiant en licence 4 en réseau et télécommunications. 
je suis en fin de cycle, je prépare mon mémoire fin d’étude dont le projet est l’étude et mise en place d’un SOC pour une entreprise qui est dans mon pays Guinée Conakry. 
Mais depuis des mois je cherche a installer sécurity onion dans mon Vmware que je n’arrive pas jusqu’à  l’heure où je vois écrit . 
J’ai eu l’idée de faire un recours chez vous pour demander de l’aide : 
Document, des liens ou des vidéos m’aideront beaucoup. 
Être assisté aussi ça me feras beaucoup plaisir. 
Pour problème que je rencontre au faite , j’ai téléchargé l’iso de security onion dans  leur site, j’ai commencé l’installation jusqu’à la fin en m’attribuant l’adresse IP que je dois taper dans le navigateur web pour ouvrir l’interface.
En tapant l’adresse dans le navigateur que ça s’ouvre pas.
Merci !

**Comment by reyesj2** [comment]
From google translate 
> Good evening everyone, I am responding in the name of Diallo Saifoulaye, a 4th degree student in networking and telecommunications.
I am at the end of my cycle, I am preparing my end-of-study thesis, the project of which is the study and implementation of a SOC for a company which is in my country Guinea Conakry.
But for months I have been trying to install security onion in my Vmware which I have not been able to achieve until the moment I see it written.
I had the idea of contacting you to ask for help:
Document, links or videos will help me a lot.
Being assisted would also make me very happy.
For the problem I encountered, I downloaded the security onion iso from their site, I started the installation until the end by assigning myself the IP address that I must type in the web browser to open the interface.
When typing the address into the browser it doesn't open.
THANKS !

If that is correct then it seems to me you may be having trouble with vmware network configuration and not necessarily Security Onion. Here are two videos that go through the VMware installation process https://www.youtube.com/watch?v=KMH7cII3mpA & https://www.youtube.com/watch?v=DQkCg5P9FSw

Note that the network adapter gets set to NAT (if you want just the host running the VM to access Security Onion) or Bridged (if devices other than the host should be able to access Security Onion).

One thing you can do to verify that the Security Onion firewall is not the problem for you is run the command 
```sudo salt-call pillar.get firewall:hostgroups:analyst``` it should return with the network or IP address of the device you're attempting to connect to the Security Onion Console with. If it does not you can add a network or ip address with the command ```sudo so-firewall includehost analyst 10.250.0.1/24 --apply```
